packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -0,0 +1,47 @@
+export type EnumPullReqReviewDecision = 'approved' | 'changereq' | 'pending' | 'reviewed'
+
+export enum PullReqReviewDecision {
```

Comment:
interfaces & types should be in interfaces.ts.. not in utils.tsx

Reply:
done